src/libs/antares/study/area/list.cpp

```diff
@@ -897,12 +897,41 @@ static bool AreaListLoadFromFolderSingleArea(Study& study,
             buffer.clear() << study.folderInput << SEP << "hydro" << SEP << "prepro";
             ret = area.hydro.prepro->loadFromFolder(study, area.id, buffer.c_str()) && ret;
         }
-        if (area.hydro.series && (!options.loadOnlyNeeded || !area.hydro.prepro)) // Series
+
+        auto* hydroSeries = area.hydro.series;
+        if (!options.loadOnlyNeeded || !area.hydro.prepro) // Series
         {
             buffer.clear() << study.folderInput << SEP << "hydro" << SEP << "series";
-            ret = area.hydro.series->loadFromFolder(study, area.id, buffer) && ret;
+            ret = hydroSeries->loadGenerationTS(area.id, buffer, study.header.version) && ret;
+
+            hydroSeries->EqualizeGenerationTSsizes(
+                area, study.usedByTheSolver, study.gotFatalError);
         }
 
+        if (study.header.version < 870)
```

Comment:
```suggestion
        if (study.header.version < 890)
```

Reply:
Still a lot of **870** in the code base, we should take care of this and turn them to **890**.